HLTrigger/Egamma/plugins/HLTDisplacedEgammaFilter.cc

```diff
@@ -40,7 +40,9 @@ HLTDisplacedEgammaFilter::HLTDisplacedEgammaFilter(const edm::ParameterSet& iCon
   inputToken_ = consumes<trigger::TriggerFilterObjectWithRefs>(inputTag_);
   rechitsEBToken_ = consumes<EcalRecHitCollection>(rechitsEB);
   rechitsEEToken_ = consumes<EcalRecHitCollection>(rechitsEE);
-  inputTrkToken_ = consumes<reco::TrackCollection>(inputTrk);
+  if (!inputTrk.label().empty()) {
```

Comment:
I suggest to add a bool config parameter named `useTrackVeto`, set to true by default via fillDescriptions, rather than giving implicit meaning to certain input-tags (in this case the empty one).

Reply:
Will do. I will make a fresh commit.